cadvisor.go

```diff
@@ -35,6 +36,7 @@ import (
 	"github.com/golang/glog"
 )
 
+var argPath = flag.String("listen_path", "", "Path to listen on (UNIX socket), defaults to empty (use TCP instead)")
```

Comment:
@timstclair Just to be sure I'm understanding you correctly, is this (the `--listen` flag) something you'd expect me to post as part of this patch, or something you'd like to look at later?

Cheers,